api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -64,23 +64,11 @@ message AccessListSpec {
   // title is a plaintext short description of the access list.
   string title = 8;
 
-  // membership defines how list membership is applied. There are two possible
-  // values:
-  //   `explicit` (default): To be considered ag member of the access list, a user
-  //                         must both meet the `membership_requires` conditions AND be
-  //                         explicitly added to the list.
-  //   implicit: Any user meeting the `membership_requires` conditions will automatically
-  //             be cosidered a member of this list.
-  string membership = 9;
-
-  // ownership defines how list ownership of this list is determined. There are
-  // two possible values:
-  //   `explicit` (default): To be considered an owner of the access list, a user
-  //                         must both meet the `ownership_requires` conditions AND be
-  //                         explicitly added to the list.
-  //   implicit: Any user meeting the `ownership_requires` conditions will automatically
-  //             be considered an owner of this list.
-  string ownership = 10;
+  // membership_mode defines how list membership is applied.
+  Inclusion membership_mode = 11;
```

Comment:
I thought the membership was new, but wasn't aware of the membership/ownership fields under AccessListSpec.

This is now a breaking change, so should we keep the old fields and "translate" as appropriate? What is the impact of the changes in existing clients and deployments?

Alternatively I would understand if you'd rather not touch these now.

Reply:
Yeah, it's new in `AccessListMember`, but there is a similar concept in `AccessList` and we should handle it the same way in both cases. There should be no deployments with anything other than "Explicit", as there is currently no way for a user to create an Implicit AccessList, so I think just retiring them will be fine.

I'm actually tempted to just revert the lot to using strings.

Reply:
I've reverted the enum stuff on the protobuf side, and left it on the Go side.